lib/ruy/conditions/day_of_week.rb

```diff
@@ -4,34 +4,40 @@
 module Ruy
   module Conditions
 
-    # Expects that a Time object's date corresponds to a specified day of the week.
+    # Expects that a Time object's date corresponds to a specified day of
+    # the week
+    #
     class DayOfWeek < Condition
       DAYS_INTO_WEEK = %w(sunday monday tuesday wednesday thursday friday saturday)
 
-      attr_reader :day, :attr, :tz_identifier
+      attr_reader :day, :key, :tz_identifier
 
       # @param day
-      # @param attr
+      # @param key
       # @param tz_identifier
-      def initialize(day, attr, tz_identifier = 'UTC')
+      # @example to check that the day in 'time' is wednesday
+      #   DayOfWeek.new(:wednesday, :time)
+      # @example to check that the day in 'time' is wednesday in CST
+      #   DayOfWeek.new(:wednesday, :time, 'CST')
+      def initialize(day, key, tz_identifier = 'UTC')
         super
         @day = day
-        @attr = attr
+        @key = key
         @tz_identifier = tz_identifier
       end
 
       def ==(o)
         o.kind_of?(DayOfWeek) &&
           o.day == @day &&
```

Comment:
We want it to be consistent with all the other files where there are too many keys to make it a one liner.